### ISSUE TITLE
Content request: hyperdrives and fuel pods

### ISSUE THREAD

**Issue by Nescio0** [endless-sky/endless-sky#3534]
Currently about all species use human hyperdrives, even though it is implied they developed independently of humans. It would be great if there were Coalition, Hai, Korath, Remnant, and Wanderer hyperdrives as well, so they no longer have to use the human version.

**Comment by KiLEdEnNis** [comment]
that has been an idea for long, but the problem is there may be 1 that's straight-up better; the only stats to really balance it around is fuel usage, jump speed and the size
apart from that I'm not sure if it got a "no" or a "someday"

**Comment by Nescio0** [comment]
> fuel usage, jump speed and the size

That should give plenty of possibilities, e.g. human as it is right now, Hai 10% smaller, Korath +100 fuel capacity, Remnant 10% faster, Wanderer 10% less fuel consumption. And perhaps one hyperdrive would consume some energy or produce heat.
And even if they would be statistitically identical, having different icons and descriptions for different species would add flavour to the game.

**Comment by Amazinite** [comment]
#1455

**Comment by KiLEdEnNis** [comment]
MZ, 28 August 2016

> I'm trying to avoid adding even more ways where, when outfitting a new ship, you have to go around and collect the best outfit of a particular kind from each species - e.g. Korath cooling plus Deep atomic thrusters, etc. So I'm not sure I like the idea of the hyperdrive becoming one more thing that is different for every species.

**Comment by Brick63** [comment]
i would say that you could just have the hyperdrives be different sprites instead of different stats but that would just be extra fluff

**Comment by endless-sky** [comment]
The basic idea is that all hyperdrives are interchangeable, in the same way that one species' cargo expansion behaves no differently from another's. Note that if there were many different hyperdrive variants, missions that are gated on what sort of drive you have would become a lot more complicated.

**Comment by KiLEdEnNis** [comment]
> If they're all interchangeable it's still possible to have statistically identical hyperdrives with different sprites, names, and descriptions for different species.

"with different names"

I'll redirect ya to MZ:
> Note that if there were many different hyperdrive variants, missions that are gated on what sort of drive you have would become a lot more complicated.

to create different species' hyperdrive images for flavor would require new outfits which, like MZ said, make these missions more complicated

**Comment by Nescio0** [comment]
> remember the Unfettered Jump Drive donations? 

Those do not *require* an hyperdrive, they *grant* one. Having multiple versions won't make this any bit more complex.

Anyway, how about giving each species its own fuel pod? E.g.:
* `Coalition: 36000 credits, 15 mass, 200 fuel;` "Coalition fuel pods are twice as large as their human counterparts, allowing Coalition to travel further, or go somewhere and return immediately without refuelling."
* `Remnant: 23000 credits, 9 mass, 120 fuel;` "Remnant fuel pods are somewhat larger than ordinary human fuel pods, because cloaking also consumes fuel."
* `Human: 20000 credits, 8 mass, 100 fuel;` "A fuel pod increases your fuel capacity, allowing your ship to make one additional hyperspace jump before it needs to land and refuel."
* `Hai: 11000 credits, 4 mass, 50 fuel;` "The Hai highly value life. Although fuel pod accidents are extremely rare, two smaller containers are safer than one large pod."
* `Wanderer: 6000 credits, 2 mass, 25 fuel;` "Wanderer fuel pods are tiny compared to those of other species. This offers greater flexibility for outfitting their ships, especially in combination with their very efficient ramscoops."

Wanderer and Hai versions are exactly as mass efficient as the human fuel pod, but their smaller size offers more flexibility (and are useful if you're using a scram drive). Remnant and Coalition fuel pods are slightly more mass efficient, but this is off-set by their less convenient fuel amount (R) and size (C). This ensures all versions remain relevant in the late game and none is “straight-up better”. 
Giving each species their own fuel pod adds some flavour, making the game more enjoyable for players who care about such minor details.

**Comment by Tadrix** [comment]
> because cloaking also consumes fuel

Shove one more fuel pod, or one more emergency ramscoop into the ship.

> Hai: 11000 credits, 4 mass, 50 fuel;
> Wanderer: 6000 credits, 2 mass, 25 fuel;

Would someone buy it? With these prices, an extra ramscoop is an obviously better choice. And diversity for diversity's sake isn't a good thing.

Now, larger Coalition fuel pods would be more logical (given distances involved). But then it would need to be decided if it should be T1 tech (15 outfit space is ok) or 1,5 T (14 t), or T2 tech (more like 13 t), or higher (12 t?).

Also, all distances in Hai space are relatively short, so don't really _need_ fuel pods. They are there because human merchants use them (so they most probably are just ordinary human fuel pods).

**Comment by endless-sky** [comment]
This doesn't seem like something that would actually improve game play at all. It makes sense for species to have different engines, for example, because engines have enough different traits (energy use, heat production, thrust, size, etc.) that each species can have engines that are good in some ways and bad in others. But a fuel pod's *only* characteristic is how much fuel it provides per outfit space, so there's no way to differentiate them. So, it's far better to just treat them as if they're all interchangeable rather than having multiple variants for no reason. And, the same applies to hyperdrives.

**Comment by RedLucidity** [comment]
I was thinking, this could be easily implemented with missions accessing a tag; Hyperdrive vs. a name hyperdrive. As such any hyperdrive would fulfill the mission and would allow for a variety of species specific tech, that belong to various categories of outfits. So, "Wanderer hyperdrive" with tag "hyperdrive" will still satisfy any criteria a hyperdrive be installed. If also the current way is left in place, then specific items could be identified or within a category that was tagged. 
The outfit tags  could also allow for unique fuel pods and other outfit variants without necessarily expanding complexity unless desired and increase the depth of flavor for each species. 

As an example: to satisfy this mission a Wanderer Hyperdrive must be installed. 
1) Say hyperdrives have a story driven signature, that can identify it as a plantary system friendly craft. Not having one in the fleet or on the flag ship could cause an international incident if the wrong craft's drives signature entered into system. 

Example with added game complexity: A Wanderer Hyperdive is attuned to specific pug built hyperspace links, versus "Milky Way" designed hyperspace links. As such having a jump drive can be even more useful, but also attenuating different hyperspace links that may or may not be more effective for intergalactic routing.  

Having such could allow for shipping missions that can only be successful due to speed and hyperspace route access versus limited fuel economy. Also jobs that have additional rewards for prompt or early delivery. All of which could be effected by fuel capacity and hyperdrive speed as an aspect of success.
Sorry tangent..

Adding tags to outfits that are accessible to mission parameters could allow for expanded mission depth and allow for flavor depth if not also ship tweaking.

**Comment by DingusShingleton** [comment]
Is there any way to have species-specific outfits in name and sprite only? As in, the human and hai have their own fuel pods, sure, but in cost, functionality, and every other way they are interchangeable. It would do much for immersion, but methinks that the amount of effort required wouldn't really be worth it.

**Comment by Amazinite** [comment]
>Is there any way to have species-specific outfits in sprite only?

Sounds like outfit variants. #1998 #927

**Comment by Pointedstick** [comment]
Duplicate of https://github.com/endless-sky/endless-sky/issues/1455, which was vetoed.

I'd be in favor of species-specific sprites for common outfits though. That could be really fun.
